plugins/connection/docker.py

```diff
@@ -57,6 +57,7 @@
 from ansible.module_utils._text import to_bytes, to_native, to_text
 from ansible.plugins.connection import ConnectionBase, BUFSIZE
 from ansible.utils.display import Display
+from ansible.plugins.shell.powershell import _parse_clixml
```

Comment:
I think it would be better to wrap that import in try/except, similarly to how it is described here: https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/dev_guide/testing/sanity/import.rst

I'd prefer that if this import ever breaks (for whatever reason, could also be `xml` not being available that's imported by the `powershell` shell plugin, or the shell plugin being moved to a collection that's not a dependency of `community.docker`), this won't break the docker connection plugin for everyone (but only for Windows containers).

Reply:
I moved the import right at the position where the `_parse_clixml` it's required.
![image](https://user-images.githubusercontent.com/14177833/109700887-a2b2ca80-7b92-11eb-839a-8bc90311fa87.png)